### DISCUSSION TITLE
在不借助三方dns工具的情况下，mihomo中应该如何配置某条域名只返回ipv4

### DISCUSSION THREAD

**Discussion by EdWard0x** [nikkinikki-org/OpenWrt-nikki#353] [Q&A]
如题，在虚空终端中没翻到相关的dns设置

**Comment by morytyann** [comment]
hosts？

**Reply by EdWard0x** [reply]
这怕是不行吧？hosts有ipv4关键字吗？我是要访问claude，因为v6是直连的，被识别为大陆。之前用隔壁home proxy是可以设置dns服务器专门解析claude的域名，mihomoTproxy没找到在哪

**Reply by morytyann** [reply]
你这样设置想解决什么问题？

**Reply by EdWard0x** [reply]
因为Claude链接的时候会使用IPV6链接那，我v6是直连出网的，所以会被BAN，于是我之前才用的这种方法解决的。现在转到mihomo，找不到相关设置了（代理V6这种情况的话就不考虑了，我选择直连出网肯定是在外有需求的）

**Reply by morytyann** [reply]
关闭dns.ipv6，或者用fakeip模式。不过好像都不能满足你的需求，其他的办法我想不到了。

**Reply by EdWard0x** [reply]
感觉应该是有办法，只是没找到相关设置，等等看有没有大佬提出方案吧
